webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -806,10 +818,21 @@ <h2>Thread {{ thread.thread }}{% if thread.thread_name %}, Name: {{ thread.threa
           </div>
           <!-- /details -->
 
-          <div id="metadata" class="ui-tabs-hide">
+          <div id="annotations" class="ui-tabs-hide">
+            <div>
+              These are the crash annotations from the submitted crash report. You
+              can see the annotations and descriptions at <a href="https://hg.mozilla.org/mozilla-central/file/tip/toolkit/crashreporter/CrashAnnotations.yaml">https://hg.mozilla.org/mozilla-central/file/tip/toolkit/crashreporter/CrashAnnotations.yaml</a>.
```

Comment:
I think telling people where the data comes from helps bridge the "what is this and how is it calculated" gap we have now. I think we can do a lot more on this front, though.